lib/charms/postgresql_k8s/v0/postgresql.py

```diff
@@ -777,9 +777,10 @@ def list_valid_privileges_and_roles(self) -> Tuple[Set[str], Set[str]]:
                 "superuser",
             }, {role[0] for role in cursor.fetchall() if role[0]}
 
-    def set_up_database(self) -> None:
+    def set_up_database(self, temp_location: Optional[str] = None) -> None:
```

Comment:
Divergence from k8s 16/edge, but it doesn't seem necessary for 14/edge charms and we should be using the v1 lib for k8s 16/edge eventually. Should I keep the new param?

Reply:
We can drop this new parameter.